tests/integration/rooms.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { setup, clearDb, get, post } from '../utils';
+import { getRepository } from 'typeorm';
+import * as HttpStatus from 'http-status-codes';
+
+import { Room } from '../../src/entity/Room';
+
+describe('Rooms Routes', () => {
+  beforeAll(setup);
+  beforeEach(clearDb);
+
+  describe('GET /rooms/:id', () => {
+    it('should return a room', async () => {
+      const room = await getRepository(Room).save({
+        name: 'foo',
+        description: 'bar',
+        options: [{ label: 'option label', value: 'option value' }],
+      });
+      const res = await get(`/rooms/${room.id}`);
+      expect(res.status).toEqual(HttpStatus.OK);
+      expect(res.body).toHaveProperty('data', { ...room });
```

Comment:
We should cover the room object more extensively since it's one of the core entities of this system.

Reply:
I agree, I was just lazy jajaja, should we add an expectation for each expected attribute? here we are expecting to receive an object with the same shape we sent to the server, the only thing we are not checking is that the `id` columns are sent (for the room and each of the options)